gumby/modules/tribler_module.py

```diff
@@ -262,8 +262,8 @@ def write_overlay_statistics(self):
 
         # Write bandwidth statistics
         with open('bandwidth.txt', 'w') as bandwidth_file:
-            bandwidth_file.write("%d,%d" % (self.session.ipv8.endpoint.bytes_up,
-                                            self.session.ipv8.endpoint.bytes_down))
+            bandwidth_file.write("%d,%d" % (self.session.ipv8.endpoint.interfaces["UDPIPv4"].bytes_up,
```

Comment:
Hmm.. this is not pretty. I'll add the `bytes_up` and `bytes_down` properties to the `DispatcherEndpoint`.

Reply:
If you want to wait for https://github.com/Tribler/py-ipv8/pull/837, you don't need this change.

Reply:
If you use the latest IPv8 you can drop commit d6df84dbca2d65f74fd9a879e3d46b9bf692f28d from this PR.

Reply:
Fixed 👍

Reply:
If at all possible I'd like IPv8 to good looking, complete and easy to use 😄